Azure/sonic-buildimage
# Add service to restore TACACS from old config

#### Why I did it
In upgrade scenarios, where config_db.json is not carry forwarded to new image, it could be left w/o TACACS credentials.
Added a service to trigger 5 minutes after boot and restore TACACS, if /etc/sonic/old_config/tacacs.json is present.

#### How I did it
By adding a service, that would fire 5 mins after boot.
This service apply tacacs if available.

#### How to verify it
Upgrade and watch status of tacacs.timer & tacacs.service
You may create /etc/sonic/old_config/tacacs.json, with updated credentials
(_before 5mins after boot_) and see that appears in config & persisted too.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
